src/app.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import 'babel-polyfill';
```

Comment:
if you set it up in webpack, you don't need it here. Or if you import it here, you don't need it in webpack. 
Please, remove extra definition here or in webpack config.